src/main/resources/application.yml

```diff
@@ -78,5 +82,9 @@ management:
     info:
       cache:
         time-to-live: 2000ms
+  info:
+    env:
+      enabled: true
```

Comment:
This is a bit cryptic as well.. is this for the health endpoints?

Reply:
So this is for the info end point - Spring Boot 2.6 has removed some of the data the info endpoint returns by default - this makes sure we can still get the app name